1.9/administration/installing/custom/configuration-parameters.md

```diff
@@ -200,9 +200,9 @@ This parameter specifies whether to enable the DC/OS proxy.
 *  `use_proxy: 'false'` Do not configure DC/OS [components](/docs/1.9/overview/architecture/components/) to use a custom proxy. This is the default value.
 *  `use_proxy: 'true'` Configure DC/OS [components](/docs/1.9/overview/architecture/components/) to use a custom proxy. If you specify `use_proxy: 'true'`, you can also specify these parameters:
     **Important:** The specified proxies must be resolvable from the provided list of [resolvers](#resolvers).
-    *  `http_proxy: <your_http_proxy>` This parameter specifies the HTTP proxy.
-    *  `https_proxy: <your_https_proxy>` This parameter specifies the HTTPS proxy.
-    *  `no_proxy: - <ip-address>` This parameter specifies YAML nested list (-) of addresses to exclude from the proxy.
+    *  `http_proxy: http://<user>:<pass>@<proxy_host>:<http_proxy_port>` This parameter specifies the HTTP proxy.
```

Comment:
@vishnu2kmohan please make this update in https://github.com/mesosphere/dcos-docs-enterprise too. this particular content is not shared betweens sites.